main.go

```diff
@@ -459,6 +479,32 @@ func (o *Options) runServerMode(pinnedDateTime *time.Time) error {
 		pinnedDateTime,
 	)
 
+	// Do an immediate metrics update
+	err = metrics.RefreshMetricsDB(dbc, util.GetReportEnd(pinnedDateTime))
+	if err != nil {
+		log.WithError(err).Error("error refreshing metrics")
+	}
+
+	// Refresh our metrics every 20 minutes:
+	ticker := time.NewTicker(20 * time.Minute)
+	quit := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				log.Info("tick")
+				err := metrics.RefreshMetricsDB(dbc, util.GetReportEnd(pinnedDateTime))
+				if err != nil {
+					log.WithError(err).Error("error refreshing metrics")
+				}
+			case <-quit:
+				ticker.Stop()
+				return
+			}
+		}
+	}()
```

Comment:
Will we end up in a weird state if the replicas are refreshing metrics at different times? Let's say we have 2 sippy containers that start 10 minutes offset each other.  Couldn't we end up in a situation where we flap on alerts briefly when one instance reached an alerting threshold, and another didn't because it didn't refresh it's metrics yet?

There's https://github.com/krayzpipes/cronticker but it hasn't been updated in years.

Maybe it's possible to make the metrics ingress endpoint sticky to only ever hit one of the endpoints?

Reply:
Hmm that's a very good question. 

I think we possibly best mitigate with the "for:" lines in the alert config, saying to only fire if the condition has been true for at least 10 minutes? We have some set to 1s, which we could change now that this would be in play. Maybe we set every pod to refresh metrics every 5 mins, and use at least 10 mins for firing alerts to give us a good buffer. I think metrics refreshes are pretty quick. Thoughts?

Reply:
I think that would work

Reply:
I will go bump those thresholds for the sister PR to this one so it merged together.

Reply:
Changed to 5m here.